### PR TITLE
feat: add aws sdk types dependency (to be used in function templates)

### PR DESCRIPTION
**Pull request by rsioss**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -67,7 +67,7 @@
     "cli-select": "1.1.2",
     "dotenv": "16.3.1",
     "esbuild": "0.18.10",
-    "eslint": "8.43.0",
+    "eslint": "8.45.0",
```
Comment (rsioss) [inline package.json:70]: unrelated change that should address https://github.com/Stedi-Demos/integrations-sdk/security/dependabot/5

**Review by BDQ — APPROVED** [review]
